react/modules/imageAsList.tsx

```diff
@@ -27,6 +28,7 @@ export const getImagesAsJSXList = (
         maxHeight={height}
         width={width}
         experimentalPreventLayoutShift={experimentalPreventLayoutShift}
+        preload={preload}
```

Comment:
This will force all images on a `list-context.image-list` to be preloaded. I don't think that's intended even if users set it to true, right?